data/pathfinder/paizo/roleplaying_game/ultimate_magic/um_templates.lst

```diff
@@ -35,3 +35,5 @@ Starting Gold UM ~ Maximum	VISIBLE:NO																																	KIT:1|Star
 Starting Gold UM ~ Average	VISIBLE:NO																																	KIT:1|Starting Gold UM ~ Average
 
 Magus Spellbook			VISIBLE:NO																																	KIT:1|Magus Spellbook
+
+Mindchemist Language	VISIBLE:NO			LANGBONUS:ALL
```

Comment:
Ah... Right! Did the same thing in ACG with the Inspired Chemist.
 

    On Sunday, April 24, 2016 12:31 PM, Andrew Maitland <notifications@github.com> wrote:
 

 In data/pathfinder/paizo/roleplaying_game/ultimate_magic/um_templates.lst:> @@ -35,3 +35,5 @@ Starting Gold UM ~ Maximum	VISIBLE:NO																																	KIT:1|Star
>  Starting Gold UM ~ Average	VISIBLE:NO																																	KIT:1|Starting Gold UM ~ Average
>  
>  Magus Spellbook			VISIBLE:NO																																	KIT:1|Magus Spellbook
> +
> +Mindchemist Language	VISIBLE:NO			LANGBONUS:ALL
Unless they can learn secret languages, you may want to use the standard method of "TYPE=Spoken"—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub